extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -50,11 +45,14 @@ amp-sticky-ad {
 
 .-amp-sticky-ad-close-button {
   position: absolute;
+  display: block;
   height: 32px;
   width: 32px;
   background-color: #fff;
   color: grey;
-  font-size: 100%;
+  font-size: 150%;
```

Comment:
Since you define the fixed width and height, you can also define a fixed `px` font-size. And ideally you'd also want to set line-height to the same value in that case.